classes/security/authorization/internal/SubmissionFileRevisionStageRequiredPolicy.inc.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file classes/security/authorization/internal/SubmissionFileRevisionStageRequiredPolicy.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionFileRevisionStageRequiredPolicy
+ * @ingroup security_authorization_internal
+ *
+ * @brief Submission file policy to ensure that we have a file at revision stage.
+ *
+ */
+
+import('lib.pkp.classes.security.authorization.internal.SubmissionFileBaseAccessPolicy');
+
+class SubmissionFileRevisionStageRequiredPolicy extends SubmissionFileBaseAccessPolicy {
```

Comment:
I wonder if I can reduce all of these polices to the new and generic one:
-- SubmissionFileSubmissionStageRequiredPolicy
-- SubmissionFileViewableReviewerResponseRequiredPolicy
-- SubmissionFileRevisionStageRequiredPolicy
The second one has also a user check, s. https://github.com/pkp/pkp-lib/blob/master/classes/security/authorization/internal/SubmissionFileViewableReviewerResponseRequiredPolicy.inc.php#L38-L40.
@asmecher, do you think I can add that user check also for the other two policies, i.e. may I add the user check in that new, generic class? Or may I leave it out, also for the second one, SubmissionFileViewableReviewerResponseRequiredPolicy?